### PR TITLE
Modified empty_subscription_feed_subtitle to improve discoverability of the subscription import/export menu

### LINKED ISSUE #8504: Discoverability of the subscription import/export menu
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

The "Nothing here but crickets" in the Subscriptions fragment should be replaced with a "You can import and export subscriptions from the three-dot menu", or something similar, in order to have better discoverability. Note that this means that that message will show up only when there are no subscriptions, but I think that's fine. For reference, see #7458.

### PR DESCRIPTION
**Pull request by Poslovitch**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Changed the default value for `empty_subscription_feed_subtitle` string key
- Removed all translations for this key (as requested in #8508)

#### Before/After Screenshots/Screen Record

- Before:

- After: 

#### Fixes the following issue(s)

- Fixes #8504 

#### APK testing 

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -747,4 +746,5 @@
     <string name="select_quality_external_players">Select quality for external players</string>
     <string name="unknown_format">Unknown format</string>
     <string name="unknown_quality">Unknown quality</string>
+    <string name="empty_subscription_feed_subtitle">You can import and export subscriptions from the three-dot menu</string>
```
Comment (litetex) [inline app/src/main/res/values/strings.xml:749]:
```suggestion
    <string name="empty_subscription_feed_subtitle">You can import and export subscriptions using the three-dot menu</string>
```

**Review by litetex — CHANGES_REQUESTED** [review]
I think we need a few changes:
1. Somehow "empty_subscription_feed_subtitle" is used inside ``list_empty_view.xml`` which is used nearly everywhere and not just in the subscription area:
grafik
2. The layout is not correctly aligned
3. See the comment below

**Comment by foss-enjoyer** [comment]
That's way too long. "Import & export subscriptions from 3-dot menu." would fit better.

**Comment by Poslovitch** [comment]
I'm getting back on working on this PR. Please pardon the long delay.

**Comment by Stypox** [comment]
Since I couldn't push to this PR (you probably forbade pushes to this branch by maintainers), I opened #9501 instead. Thanks anyway!

**Pull request closed without merging**